pennylane_qiskit/qiskit_device2.py

```diff
@@ -538,7 +546,7 @@ def _execute_runtime_service(self, circuits, session):
 
         program_inputs = {
             "circuits": compiled_circuits,
```

Comment:
Does the compilation of the circuit use Qiskit's transpilation?

Reply:
Not at the moment. This is a 1 to 1 conversion of a circuit written in Qiskit to a circuit written in Pennylane. 

There should be an additional, separate step like transpile_qiskit_circuit that takes the qiskit_circ and transpiles it according to the specifications of the backend. It should be a separate step since this is only necessary if you want to access the hardware.